sentry_sdk/integrations/django/asgi.py

```diff
@@ -135,10 +135,14 @@ async def sentry_patched_get_response_async(self, request):
 def patch_channels_asgi_handler_impl(cls):
     # type: (Any) -> None
 
-    import channels  # type: ignore
+    try:
+        import channels  # type: ignore
+    except ImportError:
+        channels = None
+
     from sentry_sdk.integrations.django import DjangoIntegration
 
-    if channels.__version__ < "3.0.0":
+    if channels is not None and channels.__version__ < "3.0.0":
```

Comment:
Not sure if this doesn't change the meaning of the `else` in this condition. Before this change, it looks like the else was meant for folks having channels installed but version 3+. Now, it'll also execute if channels is not installed at all. Is this ok? Maybe we need an early return before this whole condition if channels is `None` instead?

Reply:
Kind of. the problem is that `patch_channels_asgi_handler_impl` called in `_patch_channels` with a non channels class. So I guess I should make the whole thing better...

Reply:
i guess a fix that I did earlier just made everything less clean...

Reply:
updated the code, this is outdated now